### DISCUSSION TITLE
LEO Metadata files

### DISCUSSION THREAD

**Discussion by Bemtgenj** [groops-devs/groops#133] [Q&A] [answered]
Hello,

 The Leo Metadata part of the kinematic orbit determination for LEO satellites StationInfoCreate script uses the following files: temporal.txt, antennas.txt and center of gravity.txt. Is here somewhere an explanation of the content of the files and what the numbers represent / in what reference frame the numbers are? I think i have an understanding for the tempoal.txt numbers: distance of antenna frame from the satellite reference frame (origin of ARF vs origin of SRF) and then if the antennas have a rotation in the frame; centerofgravity.txt - change in mass over time and center of gravity distance from SRF; antennas.txt - i am not exactly sure what distances / positions these are (ACV..). I want to make sure i set the input files up correctly for the script.

Thank you for the help

Cheers
Jean

**Comment by Bemtgenj** [comment]
Add on

I know that Barbara had already given some feedback on this question in one of the precious topics but i guess now doing another LEO mission (and not the Sentinel example anymore) i am still not 100% sure yet on how to go about it

TXS for the help

Cheers
Jean

**Comment by bsuesser-rechberger** [comment, marked as answer]
Hello, 

- temporal.txt file:  includes the coordinates of the GPS antenna(s) in satellite reference frame (SRF), which are described in the satellite owner specifications, and the rotation angles from SRF into antenna reference frame (ARF).  GROOPS requires the ARF in NEU (north-east-up) system
- antennas.txt: includes the phase center offsets (ARF)
- centerOfGravity: includes CoM coordinate changes due to mass loss

Best regards,
Barbara

**Comment by Bemtgenj** [comment]
Hello Barbara,

txs for the feedback and confirmation of the different input to the files

Cheers
Jean
